### ISSUE TITLE
Website httrack.com does not mention releases 3.49.3 and 3.49.4?

### ISSUE THREAD

**Issue by hartwork** [xroche/httrack#243]
Hi @xroche,

I've been maintaining the Gentoo package for httrack since 2012 and we've been in touch through mail before a few times.
I just learned that there are releases 3.49.3 and 3.49.4 only through mail from [repology.org](https://repology.org/project/httrack/information), the website at https://www.httrack.com/ does not mention either release, I found this repo using Google eventually.
Are there any plans to get https://www.httrack.com/ back up to date or to make it point here somewhere super visible?

Thanks and best, Sebastian

**Comment by xroche** [comment]
Currently I'm only releasing for the Linux version (which is far less visible than the Windows one); there are no plans for a full release yet (due to time constraints)

The most up-to-date place is on the git repository (currently with the tag 3.49.4)

I will try to update the version at the download section though (https://www.httrack.com/page/2/fr/index.html)

**Comment by Neustradamus** [comment]
Linked to:
- https://github.com/xroche/httrack/issues/243
- https://github.com/xroche/httrack/issues/249
- https://github.com/xroche/httrack/issues/268

**Comment by hartwork** [comment]
No interest apparently, closing to keep my https://github.com/issues clean…

**Comment by Neustradamus** [comment]
@xroche: It is possible to create Windows 3.49.5 release builds?
The current is very old and is 3.49-2 (2017-05-20), more than 7 years old.

**Comment by jddcef** [comment]
> [@xroche](https://github.com/xroche): It is possible to create Windows 3.49.5 release builds? The current is very old and is 3.49-2 (2017-05-20), more than 7 years old.

Yes, in github, one can run this Actions or continuous.yml type files that can build the exe on their servers!
For example, I forked the winhttrack repo, and updated the continous.yml build file to fix the github version deprecations and allow it to be run manually: https://github.com/jddcef/httrack-windows/commit/4d7bb517bf5558c61cd921aa23ffb87fc44be8f

And then it's easy to run the build https://github.com/jddcef/httrack-windows/actions/runs/14228565555 here's my output run.
